ffi/diplomat/diplomat_gen_all.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
-diplomat-tool c ./c/include
-diplomat-tool cpp ./cpp/include
-diplomat-tool js ./wasm/lib/ ./wasm/docs.rst
+diplomat-tool c ./c/include --docs-base-urls=icu:https://unicode-org.github.io/icu4x-docs/doc/ --docs-base-urls=fixed_decimal:https://unicode-org.github.io/icu4x-docs/doc/ --docs-base-urls=icu_plurals:https://unicode-org.github.io/icu4x-docs/doc/ --docs-base-urls=icu_properties:https://unicode-org.github.io/icu4x-docs/doc/ --docs-base-urls=icu_codepointtrie:https://unicode-org.github.io/icu4x-docs/doc/ --docs-base-urls=icu_uniset:https://unicode-org.github.io/icu4x-docs/doc/ --docs-base-urls=icu_provider_blob:https://unicode-org.github.io/icu4x-docs/doc/ --docs-base-urls=icu_provider_fs:https://unicode-org.github.io/icu4x-docs/doc/ --docs-base-urls=icu_segmenter:https://unicode-org.github.io/icu4x-docs/doc/ --docs-base-urls=icu_provider:https://unicode-org.github.io/icu4x-docs/doc/
```

Comment:
issue: I'd rather not do this, this is brittle

As I'd proposed [here](https://github.com/rust-diplomat/diplomat/pull/136#discussion_r835496570), can we have a single "default" parameter for the doc base URL? we could continue to allow for passing in additional stuff. And we can special case `docs.rs` to mean "link to docs.rs with the correct format"